### PR TITLE
Use Koin from mavenCentral()

### PR DESCRIPTION
**Pull request by hannesa2**
It's one more step closer to remove `jcenter()`

This is remaining 
image

### CODE REVIEW

**Comment by jesmrec** [comment]
QA: regression

File: owncloudApp/build.gradle [hunk 1]
```diff
@@ -47,7 +47,7 @@ dependencies {
     implementation "androidx.room:room-runtime:$roomVersion"
 
     // Koin dependency injector
-    implementation "io.insert-koin:koin-android:3.0.2"
+    implementation "io.insert-koin:koin-android:$koin"
```
Comment (hannesa2) [inline owncloudApp/build.gradle:50]: What's the point for a variable for a usage of just _**one**_ place ?
It only prevents from usage of Intent action in Android Studio

**Comment by hannesa2** [comment]
Your commit message is to long 
image

image

File: owncloudApp/build.gradle (hunk 1 above)
Comment (hannesa2) [reply owncloudApp/build.gradle:50]: image

A variable does not work here

**Comment by hannesa2** [comment]
Which bugs are this ? Please explain.

Why not move forward to `3.1.2` ?

**Comment by abelgardep** [comment]
> Which bugs are this ? Please explain.
> 
> Why not move forward to `3.1.2` ?

We got some problems with UI tests after upgrading to the latest version. We plan to use this PR to move Koin to mavenCentral. We will bump the version into an additional PR.

**Comment by hannesa2** [comment]
I guess you never tried out `3.1.2`

File: owncloudApp/build.gradle (hunk 1 above)
Comment (fesave) [reply owncloudApp/build.gradle]: We are going to use it in two places again, we restore the variable.

**Review by abelgardep — APPROVED** [review]
LGTM. Thanks @hannesa2 for the contribution. We will bump the version in upcoming PRs.

**Pull request merged**